### PR TITLE
SPOI-6048: test-PR-build

### PR DESCRIPTION
**Pull request by sandeepdeshmukh**
Testing to automatically trigger PR build in Jenkins.

### CODE REVIEW

File: api/src/main/java/com/datatorrent/api/Context.java [hunk 1]
```diff
@@ -302,7 +302,60 @@
...
+      * by the engine. The attribute is ignored when the module was already declared stateless through the
+      * {@link Stateless} annotation.
+      */
+     // sets this attribute to all operators in the module, not overriding if already specified in the operator by {@link Stateless}.
+     Attribute<Boolean> STATELESS = new Attribute<Boolean>(false);
+
+     /**
+      * The options to be pass to JVM when launching the operator. Options such as java maximum heap size can be specified here.
+      */
+     // DO WE NEED THIS
```
Comment (ilganeli) [inline api/src/main/java/com/datatorrent/api/Context.java]: Can we make this a TODO

File: engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java [hunk 2]
```diff
@@ -869,6 +869,39 @@ public int hashCode()
     return operator;
   }
 
+  @Override public <T extends Module> T addModule(String name, T module)
+  {
+
```
Comment (ilganeli) [inline engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java]: Nit: Extra space

File: engine/src/test/java/com/datatorrent/stram/engine/RecoverableInputOperator.java [hunk 3]
```diff
@@ -37,7 +36,7 @@
 public class RecoverableInputOperator implements InputOperator, com.datatorrent.api.Operator.CheckpointListener
 {
   public final transient DefaultOutputPort<Long> output = new DefaultOutputPort<Long>();
-  long checkpointedWindowId;
+  private long checkpointedWindowId;
   boolean firstRun = true;
```
Comment (ilganeli) [inline engine/src/test/java/com/datatorrent/stram/engine/RecoverableInputOperator.java]: Let's make all these private as well.

**Comment by tweise** [comment]
please close this PR

**Pull request closed without merging**